domain/removal/state/types.go

```diff
@@ -42,6 +42,20 @@ type entityAssociationCount struct {
 	Count int `db:"count"`
 }
 
+// entityAssociationCount holds a Count in int form and the UUID in string form
+// for the associated entity.
+type entityAssociationAliveCount struct {
```

Comment:
Not sure you're going to repurpose this, in which case the `Alive` and `NotAlive` parts could refer explicitly to units, and the `Associated` could refer to child machines.